buzsaki_lab_to_nwb/tingley_code_septal/convert_tingley_septal.py

```diff
@@ -0,0 +1,101 @@
+from pathlib import Path
+import warnings
+
+from scipy.io import loadmat
+
+from buzsaki_lab_to_nwb import TingleySeptalNWBConverter
+
+stub_test = True
+conversion_factor = 0.195  # Intan
+data_path = Path("/shared/catalystneuro/Buzsaki/TingleyD/")
+nwb_output_path = Path("/shared/catalystneuro/Buzsaki/TingleyD/nwbfiles")
+
+subject_list = ["DT2", "DT5", "DT7", "DT8", "DT9"]
+
+invalid_session = [
+    "20170411_1296um_1152um_170411_113418",  # No data
+    "20170527_1260um_1072um_merge",  # No data
+    "20170528_1332um_1108um_170528_114153",  # No data
+    "z_Intruder_test_160304_152951",  # Test
+    "z_USV_test_3612um_1360um_20160307_160307_202140",  # Test
+    "z_novel_cage_test_160227_165229",  # Test
+]
+
+session_path_list = [
+    session
+    for subject in data_path.iterdir()
+    if subject.is_dir() and subject.name in subject_list
+    for session in subject.iterdir()
+    if session.is_dir() and session.name not in invalid_session
+]
+
+# session_path_list = [Path("/shared/catalystneuro/Buzsaki/TingleyD/DT9/20170508_468um_36um_170508_102251")]
+
+counter = 0
+for session_path in session_path_list:
+    print("----------------")
+    print(session_path)
+    counter += 1
+    print(f"session {counter} of {len(session_path_list)}")
```

Comment:
```suggestion
for counter, session_path in enumerate(session_path_list):
    print("----------------")
    print(session_path)
    print(f"session {counter} of {len(session_path_list)}")
```

Reply:
Yes, the reason I did not use an enumerate was because I was planning on removing it eventually.